engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -734,7 +724,7 @@ - (BOOL)stateIsActive {
   if (flutterApplication) {
     isActive = [self isApplicationStateMatching:UIApplicationStateActive
                                 withApplication:flutterApplication];
-  } else if (@available(iOS 13.0, *)) {
+  } else {
```

Comment:
Optional: now that there is always a code path that sets this, you could collapse to initializing while declaring:

```objc
BOOL isActive = flutterApplication ? <current if path> : <current else path>
```

Reply:
Done... but even more compacted.